src/features/dataset/modal/EditDatasetTitleModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import { useDispatch } from "react-redux"
+
+import IconButton from "../../../chrome/IconButton"
+import { clearModal } from "../../app/state/appActions"
+import Button from "../../../chrome/Button"
+import TextInput from "../../../chrome/forms/TextInput"
+// import { selectIsDatasetLoading, selectTitleError } from "../state/datasetState"
+import Spinner from "../../../chrome/Spinner"
+
+export interface EditDatasetTitleModalProps {
+  title: string
+  onCommit: (title: string, commitTitle: string) => void
+}
+
+const EditDatasetTitleModal: React.FC<EditDatasetTitleModalProps> = ({ title, onCommit }) => {
```

Comment:
Is there a plan to use this in the future?

Reply:
Doubtful. I removed it